pydarn/plotting/fan.py

```diff
@@ -180,7 +179,7 @@ def plot_fan(cls, dmap_data: List[dict], ax=None,
         # Plot FOV outline
         beam_corners_aacgm_lats, beam_corners_aacgm_lons, thetas, rs, ax = \
             cls.plot_fov(stid=dmap_data[0]['stid'], date=date,
-                         ranges=ranges, boundary=boundary,
+                         ranges=ranges, boundary=boundaRy,
```

Comment:
```suggestion
                         ranges=ranges, boundary=boundary,
```
not sure why, but it worked with the typo anyway for plotting FOV or grids, it breaks for fan plots